components/FullEventCard/FullEventCard.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { StyleSheet, View, Button, Text, ScrollView, CheckBox, TouchableOpacity } from 'react-native';
+
+export default function FullEventCard() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.closeButton}><Button title='X'/></View>
+      <Text style={styles.titleText}>Volleyball</Text>
+      <Text style={styles.text}>Location: Wash Park</Text>
+      <Text style={styles.text}>Start Time: 2:00PM</Text>
+      <Text style={styles.text}>End Time: 4:00PM</Text>
+      <Text style={styles.text}>Players Attending: 4</Text>
+      <Text style={styles.text}>Max Players: 8</Text>
+      <Text style={styles.text}>Skill Level: friendly game!</Text>
+      <Text style={styles.text}>Equipment:</Text>
+        <Text style={styles.text}>net</Text>
+        <Text style={styles.text}>volleyball</Text>
```

Comment:
I think in the future we could make these dynamic to update if they have the equipment or not